filament/backend/src/vulkan/VulkanDriver.cpp

```diff
@@ -691,6 +691,11 @@ void VulkanDriver::updateIndexBuffer(Handle<HwIndexBuffer> ibh, BufferDescriptor
     scheduleDestroy(std::move(p));
 }
 
+void VulkanDriver::setTextureSwizzle(Handle<HwTexture> th,
```

Comment:
Can you add a stub for the Metal driver as well?

Reply:
there is one already in this PR